### PR TITLE
Create root user documentation

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1487

### CODE REVIEW

File: software/create-a-root-user.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+By default, the first user to log in to an Astronomer Software installation becomes a System Admin in that installation. If your organization has strong security requirements, you can create a single root user and give only this user permissions to create new System Admins on your installation. 
+
+## Create the root user 
+
+To create a root user, add the following lines to your `config.yaml` file:
+
+```yaml
+global:
+	rootAdmin:
+	    username: <your-root-user-email-address>
```
Comment (jwitz) [inline software/create-a-root-user.md:17]:
```suggestion
global:
   rootAdmin:
          username: <your-root-user-email-address>
```

File: software/create-a-root-user.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+By default, the first user to log in to an Astronomer Software installation becomes a System Admin in that installation. If your organization has strong security requirements, you can create a single root user and give only this user permissions to create new System Admins on your installation. 
+
+## Create the root user 
+
+To create a root user, add the following lines to your `config.yaml` file:
+
+```yaml
+global:
+   rootAdmin:
+          username: <your-root-user-email-address>
```
Comment (jwitz) [inline software/create-a-root-user.md:17]:
```suggestion
   rootAdmin:
          username: <your-root-user-email-address>
```

File: software/create-a-root-user.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+After saving your changes, apply the configuration change. See [Apply a config change](apply-platform-config.md).
+
+By default, Astronomer generates a password for your root user email address. This password and email address are stored as Kubernetes secrets on your Astronomer installation. 
+
+After you apply the configuration change, Helm provides a command for how to retrieve these the values of these secrets. Retrieve the password and use the new **Root Admin Login** button on the Software login page to access Astronomer Software as the root user. 
+
+## Customize the root user's password
+
+### Rotate the root user's password
```
Comment (jwitz) [inline software/create-a-root-user.md:28]: To be documented

File: software/create-a-root-user.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+After saving your changes, apply the configuration change. See [Apply a config change](apply-platform-config.md).
+
+By default, Astronomer generates a password for your root user email address. This password and email address are stored as Kubernetes secrets on your Astronomer installation. 
+
+After you apply the configuration change, Helm provides a command for how to retrieve these the values of these secrets. To log in to Astronomer Software as the root user, retrieve the password and use the new **Root Admin Login** button on the Software login page.
+
+## Customize the root user's password
+
```
Comment (jwitz) [inline software/create-a-root-user.md:27]:
```suggestion

You might want to use a custom root user password to ensure that someone who has recently left your organization no longer has access to the root user. 

```

File: software/create-a-root-user.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+global:
+   rootAdmin:
+          username: <your-root-user-email-address>
+```
+
+After saving your changes, apply the configuration change. See [Apply a config change](apply-platform-config.md).
+
+By default, Astronomer generates a password for your root user email address. This password and email address are stored as Kubernetes secrets on your Astronomer installation. 
+
+After you apply the configuration change, Helm provides a command for how to retrieve these the values of these secrets. To log in to Astronomer Software as the root user, retrieve the password and use the new **Root Admin Login** button on the Software login page.
```
Comment (kwcanuck) [inline software/create-a-root-user.md:24]:
```suggestion
After you apply the configuration change, Helm provides a command for retrieving the secret values. To log in to Astronomer Software as the root user, retrieve the password and click **Root Admin Login** on the Software login page.
```

File: software/create-a-root-user.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+---
+title: 'Create a root user on Astronomer Software'
+sidebar_label: 'Create a root user'
+id: create-a-root-user
+description: Learn how to create a root user with all possible permissions on Astronomer Software. 
+---
+
+By default, the first user to log in to an Astronomer Software installation becomes a System Admin in that installation. If your organization has strong security requirements, you can create a single root user and give only this user permissions to create new System Admins on your installation.
```
Comment (kwcanuck) [inline software/create-a-root-user.md:8]:
```suggestion
By default, the first user to log in to an Astronomer Software installation is assigned the System Admin role for the installation. If your organization has strong security requirements, you can create a single root user and allow only this user permission to assign System Admin roles on your installation. 
```

File: software/create-a-root-user.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+To configure a custom password for the root user, run the following command: 
+
+```sh
+kubectl patch secret -n <your-platform-namespace> astronomer-root-admin-credentials --type=json -p='[{ "op" : "replace" , "path" : "/data/password" , "value" : "'$(echo -n "<your-new-password>" | base64)'"}]' && kubectl create job --from=cronjob/<your-release-name>-update-root-admin-password-cronjob manual3 -n <your-platform-namespace>
+```
+
+## Limit System-level user creation
+
+A common use case for having a root user is to limit System-level user creation to only a single, tightly-controlled user. To configure your system this way, add the following lines to your `config.yaml` file:
```
Comment (kwcanuck) [inline software/create-a-root-user.md:38]:
```suggestion
A common use case for having a root user is to limit system-level user creation to a single user. To configure your system this way, add the following lines to your `config.yaml` file:
```

File: software/create-a-root-user.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+          permissions:
+            system.iam.updateSystemLevelPermissions: false
+        SYSTEM_ROOT_ADMIN:
+          permissions:
+            system.iam.updateSystemLevelPermissions: true
+```
+
+After saving your changes, apply the configuration change. See [Apply a config change](apply-platform-config.md).
+
+See [Customize role permissions](manage-platform-users.md#customize-role-permissions) for how to further customize role permissions.
```
Comment (kwcanuck) [inline software/create-a-root-user.md:55]:
```suggestion
To learn more about customizing role permissions, see [Customize role permissions](manage-platform-users.md#customize-role-permissions) .
```

File: software/install-aws-standard.md [hunk 9]
```diff
@@ -227,7 +227,17 @@ kubectl create secret generic astronomer-bootstrap \
 
 ## Step 8: Configure your Helm chart
 
-> **Note:** If you want to use a third-party ingress controller for Astronomer, complete the setup steps in [Third-Party Ingress Controllers](third-party-ingress-controllers.md) in addition to this configuration.
+:::info 
+
+To use a third-party ingress controller for Astronomer, complete the setup steps in [Third-Party Ingress Controllers](third-party-ingress-controllers.md) in addition to this configuration.
```
Comment (kwcanuck) [inline software/install-aws-standard.md:232]:
```suggestion
To use a third-party ingress controller for Astronomer, see [Third-Party Ingress Controllers](third-party-ingress-controllers.md).
```

File: software/install-aws-standard.md [hunk 10]
```diff
@@ -227,7 +227,17 @@ kubectl create secret generic astronomer-bootstrap \
...
+
+:::
+
+:::info 
+
+By default, the first user to log in to your installation is given System Admin permissions. To create a dedicated root user that's responsible for adding the first members of your organization to Astronomer, complete the setup steps in [Create a root user](create-a-root-user.md) in addition to this configuration.
```
Comment (kwcanuck) [inline software/install-aws-standard.md:238]:
```suggestion
By default, the first user to log in to your installation is assigned the System Admin role. To create a dedicated root user who is responsible for adding organization users to Astronomer, see [Create a root user](create-a-root-user.md).
```

File: software/install-azure-standard.md [hunk 11]
```diff
@@ -248,7 +248,17 @@ A few additional configuration notes:
 
 ## Step 8: Configure Your Helm chart
 
-> **Note:** If you want to use a third-party ingress controller for Astronomer, complete the setup steps in [Third-Party Ingress Controllers](third-party-ingress-controllers.md) in addition to this configuration.
+:::info 
+
+To use a third-party ingress controller for Astronomer, complete the setup steps in [Third-Party Ingress Controllers](third-party-ingress-controllers.md) in addition to this configuration.
```
Comment (kwcanuck) [inline software/install-azure-standard.md:253]:
```suggestion
To use a third-party ingress controller for Astronomer, see [Third-Party Ingress Controllers](third-party-ingress-controllers.md).
```

File: software/install-azure-standard.md [hunk 12]
```diff
@@ -248,7 +248,17 @@ A few additional configuration notes:
...
+
+:::
+
+:::info 
+
+By default, the first user to log in to your installation is given System Admin permissions. To create a dedicated root user that's responsible for adding the first members of your organization to Astronomer, complete the setup steps in [Create a root user](create-a-root-user.md) in addition to this configuration.
```
Comment (kwcanuck) [inline software/install-azure-standard.md:259]:
```suggestion
By default, the first user to log in to your installation is assigned the System Admin role. To create a dedicated root user who is responsible for adding organization users to Astronomer, see [Create a root user](create-a-root-user.md).
```

File: software/install-gcp-standard.md [hunk 13]
```diff
@@ -250,7 +250,17 @@ kubectl create secret generic astronomer-bootstrap \
...
-> **Note:** If you want to use a third-party ingress controller for Astronomer, complete the setup steps in [Third-Party Ingress Controllers](third-party-ingress-controllers.md) in addition to this configuration.
+:::info 
+
+To use a third-party ingress controller for Astronomer, complete the setup steps in [Third-Party Ingress Controllers](third-party-ingress-controllers.md) in addition to this configuration.
+
+:::
+
+:::info 
+
+By default, the first user to log in to your installation is given System Admin permissions. To create a dedicated root user that's responsible for adding the first members of your organization to Astronomer, complete the setup steps in [Create a root user](create-a-root-user.md) in addition to this configuration.
```
Comment (kwcanuck) [inline software/install-gcp-standard.md:261]:
```suggestion
By default, the first user to log in to your installation is assigned the System Admin role. To create a dedicated root user who is responsible for adding organization users to Astronomer, see [Create a root user](create-a-root-user.md).
```

File: software_versioned_docs/version-0.29/manage-platform-users.md [hunk 14]
```diff
@@ -257,6 +257,10 @@ The System Admin has the same default permissions as the System Viewer and Syste
     - Variables
     - XComs
 
+### Root user
+
+A root user has all available permissions on Astronomer Software. There can only be one root user per Software installation. See [Create a root user](create-a-root-user.md).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.29/manage-platform-users.md:262]:
```suggestion
A root user has all available permissions on Astronomer Software. There can only be one root user for each Software installation. See [Create a root user](create-a-root-user.md).
```
image

**Review by kwcanuck — APPROVED** [review]

**Pull request merged**